.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -4,3 +4,7 @@ set -e
 
 # reset the repo to the commit hash that was used to build the prebuilt Codespace
 git reset --hard $(cat ./artifacts/prebuild.sha)
+
+# add harness --no-sandbox argument if running in container; reload changes
+echo 'export NO_SANDBOX_IN_CONTAINER="--no-sandbox"' >> ~/.bashrc
```

Comment:
Can we instead do the same check that we did with `OPEN_BROWSER` and detect if we are in a docker container?

See https://github.com/dotnet/runtime/pull/64277/files#diff-741825b1eea3c2056ed23e42b350c5df41ff89bceb307db35491bdc7fdded2e7R14

That way someone doesn't need to use our `devcontainer`, nor Codespaces for this to work. If they are just in a plain Docker container running tests, it works the same.